gradle.properties

```
@@ -15,5 +15,5 @@
 #
 
 group=org.sonatype.gradle.plugins
-version=3.0.0-SNAPSHOT
+version=3.1.0-SNAPSHOT
```

There are no differences in the codebase of the plugin.
If anything, I'd suggest to only bump up the patch version.
```suggestion
version=3.0.1-SNAPSHOT
```